### PR TITLE
Material mapping onto CutoutCylinderVolume

### PR DESCRIPTION
**Pull request by Corentin-Allaire**
Added the possibility to adjust the binutility to CutoutCylinderVolume this will be necessary to mapped material onto the ATLAS calorimeter for example.

Also updated the binutility created by default in the json converter. All the phi bin use to be closed, now they are closed if eHalfPhiSector is equal to Pi.

I also noticed that CutoutCylinderVolume can only be full cylinder in phi (no eHalfPhiSector) should we change that ?

### CODE REVIEW

**Comment by paulgessinger** [comment]
I omitted the phi sector here because for the intended use case it wasn't needed, and the parametrization is complicated enough as it is. So I'd argue to keep it as is for now.

**Comment by Corentin-Allaire** [comment]
> I omitted the phi sector here because for the intended use case it wasn't needed, and the parametrization is complicated enough as it is. So I'd argue to keep it as is for now.

I completely agree, I just wanted to be sure !

File: Core/include/Acts/Utilities/BinAdjustmentVolume.hpp [hunk 1]
```diff
@@ -76,6 +80,62 @@
   return uBinUtil;
 }
 
+/// @brief adjust the BinUtility bu to the dimensions of cylinder volume bounds
+///
+/// @param bu BinUtility at source
+/// @param cBounds the Cylinder volume bounds to adjust to
```
Comment (asalzburger) [inline Core/include/Acts/Utilities/BinAdjustmentVolume.hpp:86]: Shouldn't that mention cutout cylinder volumes explicitly?

**Review by asalzburger — APPROVED** [review]
Straight forward adaption with new unit tests - looks good

File: Core/include/Acts/Utilities/BinAdjustmentVolume.hpp (hunk 1 above)
Comment (Corentin-Allaire) [reply Core/include/Acts/Utilities/BinAdjustmentVolume.hpp]: Done

**Comment by asalzburger** [comment]
Ok, let's wait for the CI and then merge this one in.

**Comment by msmk0** [comment]
@asalzburger Could you re-approve so this can get merged in?

**Review by asalzburger — APPROVED** [review]

**Pull request merged**